kernel/include/priority_q.h

```diff
@@ -199,6 +203,66 @@ static ALWAYS_INLINE struct k_thread *z_priq_dumb_mask_best(sys_dlist_t *pq)
 	}
 	return NULL;
 }
+
+#if defined(CONFIG_SCHED_MULTIQ)
+
+static ALWAYS_INLINE struct k_thread *_priq_mq_mask_best_idx(struct _priq_mq *pq,
```

Comment:
Slightly more conventional naming 
```suggestion
static ALWAYS_INLINE struct k_thread *z_priq_mq_best_mask_idx(struct _priq_mq *pq,
```